functions/test/produce.test.js

```diff
@@ -0,0 +1,81 @@
+const expect = require("expect");
+const request = require("supertest");
+
+const app = require("../index");
+const test = require("firebase-functions-test")();
+const { produce } = require("./data");
+
+let id;
+
+describe("POST /produce", () => {
+  it("create a new produce", (done) => {
+    request(app)
+      .post("/produce")
+      .send(produce)
+      .expect((res) => {
+        expect(res.text).not.toBeNull();
+      })
+      .end(function (err, res) {
```

Comment:
I think you can use the => syntax. maybe something like
```suggestion
      .end((err, res) => {
```

(I think most newer JS code doesn't require the `function` keyword)